docs/design/README.md

```diff
@@ -1,50 +1,63 @@
+## What is the purpose of improvement proposals?
 
-Writing a design document can promote us to think deliberately and gather knowledge from others to get our job done better. An excellent design document is closely related to the success of our task.
+The purpose of improvement proposals is to have a central place to collect and document planned major enhancements to TiDB projects. While GitHub Issue is still the tool to track tasks, bugs, and progress, the proposals give an accessible high level overview of the result of design discussions and proposals.
 
-## Proposal Process
+## Who should initiate the proposal?
 
-1. Before starting to write a design document, please [create a new issue](https://github.com/pingcap/tidb/issues/new/choose) for tracing the process of your design.
-2. Create a new [Google Doc](https://docs.google.com/document/u/0/) to write the design document. Please refer to this [proposal template](./TEMPLATE.md).
-3. If the design document is ready to be reviewed, please put the shared link with the `can comment` permission in the issue you've created, and one of our engineers will follow up this issue and keep it updated.
-4. If needed, we may organize a communications seminar through [Google Hangouts](https://hangouts.google.com/) to discuss your design with you and the interested friends from the community.
-5. When your design is finalized, please submit a pull request (PR) to add your new file under this directory, and put the link of your PR in the issue you've created. 
-6. Once your PR has been merged, please close the old issue.
-7. Start the implementation referring to the proposal, and create a new issue to trace the process.
+Anyone can initiate a proposal but you shouldn't do it unless you have an intention of getting the work done to implement it (otherwise it is silly).
 
-## Proposal Status
+## Process
 
-### Proposed
+Here is the process for making a proposal:
 
-- [Proposal: A new command to restore dropped table](./2018-08-10-restore-dropped-table.md)
-- [Proposal: Support SQL Plan Management](./2018-12-11-sql-plan-management.md)
+1. [Create an issue](https://github.com/pingcap/tidb/issues/new/choose) to track the proposal progress, basically track subtasks.
+2. Create a pull request with a new proposal filed under this directory named as `YYYY-MM-DD-descriptive-title-of-your-proposal.md`, with content follows the [template](./TEMPLATE.md).
+3. Request review to reviewers/committers and respond to change requests. The proposal is considered accepted with 2+ approvals from committers and no change requests from reviewers/committers. Please ask a committer to help merge the proposal on accepted.
+4. **_Please_** update this page and the index below, to reflect the current stage of the proposal. This acts as the permanent record indicating the result of the proposal (Accepted, Completed or Discarded).
```

Comment:
how about giving an example PR?

Reply:
Sure. As you can see [these lines](https://github.com/pingcap/tidb/pull/23999/files#diff-c435a658e946facf16aac9569c0c5ce284159722b837124fb3cabfe8233938a4R30-R34).